### PR TITLE
fix: the card responsiveness at medium screen (in past edition page)

### LINKED ISSUE #309: [BUG] The cards on the past edition page are displaying improperly, appearing squashed
### Describe the bug.

<h1>Description of the Bug </h1>
at the medium size screen, the card content are getting tight. and the card are turning into squashed making it look unreadable

### Expected behavior

<h1>Expected behavior</h1>
starting from the medium screen we can make the card layout (grid) to 1 making it readable and clean.

### Screenshots

<h1> ScreenShot </h1>
before changes: <br>

image
<br>after changes: <br>

image

### How to Reproduce

1. I check the screen where the responsive start
2. then i make the md grid to 1

### 🥦 Browser

Brave

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by TenzDelek**
fixes #309 

before:
image

after:
image

### CODE REVIEW

File: pages/editions/index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const PastEditions = () => {
                 <h1 className='text-5xl sm:text-4xl px-10 py-4 rounded-full bg-violet-700 text-white my-4 text-center w-fit mx-auto font-bold'>AsyncAPI Conf</h1>
             </div>
 
-            <div className='w-3/4 sm:w-10/12 my-10 mx-auto grid grid-cols-2 sm:grid-cols-1 gap-[60px]'>
+            <div className='w-3/4 sm:w-10/12 my-10 mx-auto grid grid-cols-2 md:grid-cols-1 lg:grid-cols-1 gap-[60px]'>
```
Comment (ashmit-coder) [inline pages/editions/index.js:15]: In my opinion just defining it for `lg` will work and then putting `grid-cols-1` for all other displays is better ig.

File: pages/editions/index.js
Comment (ashmit-coder) [inline pages/editions/index.js:1]: Hi @TenzDelek and @AceTheCreator , there seems to be an issue with the tailwind config file.
In my opinion `grid grid-cols-2 lg:grid-cols-1` is working inversely.

    screens: {
			    xl: { max: '1279px' },
			    lg: { max: '1118px' },
			    sm: { max: '590px' },
		    },

to correct this the suggested chabge is

    screens: {
			    xl: '1279px' ,
			    lg: '1118px',
			    sm: '590px',
		    },

If Ace permits I can generate a pr or if you want @TenzDelek you can add these changes in this pr as well.

Also make the css change as follows

    <div className='w-3/4 sm:w-10/12 my-10 mx-auto grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-2 gap-[60px]'>

Comment (ashmit-coder) [reply pages/editions/index.js:1]: @TenzDelek I have raised a pr for the config issue no need to do that, you can just make changes to your part as mentioned and merge the main branch after #314 is merged.

**Review by ashmit-coder — APPROVED** [review]
LGTM!

File: pages/editions/index.js
Comment (AceTheCreator) [reply pages/editions/index.js:1]: > Hi @TenzDelek and @AceTheCreator , there seems to be an issue with the tailwind config file. In my opinion `grid grid-cols-2 lg:grid-cols-1` is working inversely.
> 
> ```
> screens: {
> 		    xl: { max: '1279px' },
> 		    lg: { max: '1118px' },
> 		    sm: { max: '590px' },
> 	    },
> ```
> 
> to correct this the suggested chabge is
> 
> ```
> screens: {
> 		    xl: '1279px' ,
> 		    lg: '1118px',
> 		    sm: '590px',
> 	    },
> ```
> 
> If Ace permits I can generate a pr or if you want @TenzDelek you can add these changes in this pr as well.
> 
> Also make the css change as follows
> 
> ```
> <div className='w-3/4 sm:w-10/12 my-10 mx-auto grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-2 gap-[60px]'>
> ```

@ashmit-coder this is a breaking change and will require you to refactor it in the whole source code

**Review by AceTheCreator — APPROVED** [review]
LGTM!!!

**Pull request merged**